enumerables.rb

```diff
@@ -63,26 +64,26 @@ def my_none(argument = nil)
 
   def my_map
     arr = self
-    mapped_arr =[]
-      arr.my_each do |i| i
-        mapped_arr.push(yield i)
-        yield i
-      end
-      mapped_arr
+    mapped_arr = []
+    arr.my_each do |i|
+      mapped_arr.push(yield i)
+      yield i
+    end
+    mapped_arr
   end
 
   def my_inject
```

Comment:
- [ ] [`inject`](https://rubyapi.org/2.7/o/enumerable#method-i-inject) takes initial value to start with and it's optional, for example:
```ruby
# without passing initial value to inject
[5, 2, 1].inject { |result, x| result + x }
# with passing initial value to inject
[5, 2, 1].inject(0) { |result, x| result + x }
```

In case of not passing initial value, ruby will select the first item of the array to be the initial value. The current implementation breaks if the initial value is not passed. Let's add a case if the initial value is not passed.